src/actions/index.test.js

```diff
@@ -83,4 +85,30 @@ describe('actions', () => {
       });
     });
   });
+
+  describe('loadRestaurant', () => {
+    beforeEach(() => {
+      store = mockStore({});
+    });
+
+    context('with restaurantId', () => {
+      it('runs setRestaurant', async () => {
+        await store.dispatch(loadRestaurant({ restaurantId: 1 }));
+
+        const action = store.getActions()[0];
+
+        expect(action).toEqual(setRestaurant({}));
+      });
+    });
+
+    context('without restaurantId', () => {
+      it('does\'nt run any actions', async () => {
```

Comment:
```suggestion
      it("doesn't run any actions", async () => {
```

저는 차라리 쌍따옴표로 감싸주시는게 가독성이 더 좋을 것 같은데 어떠하신가요?

Reply:
피드백 감사합니다. https://github.com/CodeSoom/react-week6-assignment-1/pull/52/commits/63d0364b57d806461a3ddf3b970716758d027520 에서 수정했습니다.